include/elliptics/packet.h

```diff
@@ -1110,12 +1110,15 @@ static inline void dnet_convert_defrag_ctl(struct dnet_defrag_ctl *ctl)
 struct dnet_monitor_stat_request {
 	uint64_t	categories;
 	int			reserved_int; // reserved for packing
-	uint64_t	reserved[4];
+	uint32_t	backends_ids_size;
```

Comment:
`backends_ids_size`-> `backends_number` or `backends_num` and add comment to it.